### ISSUE TITLE
example["times"]

### ISSUE THREAD

**Issue by ryanyej** [TuSimple/centerformer#41]
In multi frame 
x_up_fuse = torch.cat((x_up, x_prev_cat), dim=1) + self.time_embedding(
            example["times"][:, :, None].to(x_up)
        ).reshape(x_up.shape[0], -1, 1, 1)

what does 'times' means?
B*K, means time interval of this frame with last frame?

Looking forward to your reply.
